README.md

```diff
@@ -160,11 +166,32 @@ dataSourceName = root:123456@tcp(db:3306)/
 
 > If you need to modify `conf/app.conf`, you need to re-run `docker-compose up`.
 
+#### create database
+
+
+
+Unlike the casdoor-all-in-one image, this image don't have a MySql database inside, which means you need to create a database named `casdoor`, before running docker.
+
+
+```bash
+
+# into database container
+docker exec -it casdoor_db_1 bash 

```

Comment:
Container names may be different for other users, and I think most developers should know how to use the database, we need not descript this.

Reply:
I am trying to solve the issue raised by this friend. And thanks for reminding me, I found the container with this name is inside the image, so I thought it was the same for everyone?
https://github.com/casbin/casnode/issues/445

Reply:
If the user does not edit the `docker-compose.yml` file, the name will indeed be this.
But if someone uses `docker stack` to run the `docker-compose.yml file, it may be another name.
I still think we only need to notify the user about the option of auto-create DB default is false.